packages/flutter/lib/src/material/tabs.dart

```diff
@@ -49,6 +49,41 @@ enum TabBarIndicatorSize {
   label,
 }
 
+/// Defines how tabs are aligned horizontally in a [TabBar].
+///
+/// See also:
+///
+///   * [TabBar], which displays a row of tabs.
+///   * [TabBarView], which displays a widget for the currently selected tab.
+///   * [TabBar.tabAlignment], which defines the horizontal alignment of the
+///     tabs within the [TabBar].
+enum TabAlignment {
+  // TODO(tahatesser): Add a link to the Material Design spec for
+  // horizontal offset when it is available.
+  // It's currently sourced from androidx/compose/material3/TabRow.kt.
+  /// If [TabBar.isScrollable] is true, tabs are aligned to the
+  /// start of the [TabBar] with a horizontal offset of 52.0 pixels.
+  /// Otherwise throws an exception.
+  ///
+  /// It is not recommended to set [TabAlignment.start] when
+  /// [ThemeData.useMaterial3] is false.
+  start,
+
+  /// If [TabBar.isScrollable] is true, tabs are aligned to the
+  /// start of the [TabBar]. Otherwise throws an exception.
+  ///
+  /// It is not recommended to set [TabAlignment.startOffset] when
+  /// [ThemeData.useMaterial3] is false.
+  startOffset,
```

Comment:
I understand the difference between start and startOffset is the 52 pixels.. but they seem backwards. Should the one with the extra pixels be startOffset?

Reply:
`startOffset` is the name I came up with, it's a start positon with an offset of 52 pixels. 

`start` for start position makes sense, we could a better name for `startOffset`

Reply:
> `startOffset` is the name I came up with, it's a start position with an offset of 52 pixels.

Yeah I think it makes sense, the docs look backwards though. `start` is documented with 52 pixels.

Reply:
Oops, thank you.